--- gcc-7.3.0/libgomp/config/linux/wait.h~	2017-01-01 12:07:43.000000000 +0000
+++ gcc-7.3.0/libgomp/config/linux/wait.h	2018-02-04 18:11:47.928564439 +0000
@@ -57,7 +57,10 @@
     if (__builtin_expect (__atomic_load_n (addr, MEMMODEL_RELAXED) != val, 0))
       return 0;
     else
-      cpu_relax ();
+      if (i < count/2) 
+	       __asm__ __volatile__("nop\nnop\nnop\nnop\n": : :"memory");
+      else
+	      cpu_relax ();
   return 1;
 }
 
