From 53c6dbba92626caa021cc264d0cd6bbc0dab25f7 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 2 May 2018 21:07:18 +0000
Subject: [PATCH] openmp vectorize v2

---
 gcc/opts.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/gcc/opts.c b/gcc/opts.c
index 33efcc0d6..35ad4fef4 100644
--- a/gcc/opts.c
+++ b/gcc/opts.c
@@ -699,6 +699,15 @@ finish_options (struct gcc_options *opts, struct gcc_options *opts_set,
 {
   enum unwind_info_type ui_except;
 
+  /* openmp to imply tree vectorize */
+  if (opts->x_flag_openmp && opts->x_optimize >=2) {
+   opts->x_flag_tree_loop_vectorize = 1;
+   opts->x_optimize = 3;
+   opts->x_flag_gcse_after_reload = 1;
+   opts->x_flag_dse = 1;
+
+  }
+
   if (opts->x_dump_base_name
       && ! opts->x_dump_base_name_prefixed)
     {
-- 
2.17.0

